### ISSUE TITLE
Wazuh Kibana app doesn't check wazuh.yml file

### ISSUE THREAD

**Issue by apandiyan** [wazuh/wazuh-kibana-app#2353]
| Wazuh | Elastic | Rev |
| ----- | ------- | --- |
| 3.13   | 7.8     | ??? |

**Description**
Custom index pattern isn't working after upgrading Wazuh from 3.12 to 3.13 and Kibana 7.6.2 to 7.8.0.

Moreover it seems like the new wazuh app binded (hard coded) only on wazuh-alerts index pattern only. Kindly find attached snippets for more information.

**Steps to reproduce**

1. Create Wazuh 3.12 instance with Kibana 7.6.2 before starting kibana change the pattern to custom at /usr/share/kibana/optimize/wazuh/config/wazuh.yml
2. Start Kibana service
3. Remove default index patterns from Kibana
4. Update Wazuh from 3.12 to 3.13 and Kibana 7.6.2 to 7.8.0 using official document given at 
https://documentation.wazuh.com/3.13/upgrade-guide/upgrading/latest_wazuh3_minor.html#upgrading-latest-minor
https://documentation.wazuh.com/3.13/upgrade-guide/upgrading-elastic-stack/elastic_server_minor_upgrade.html
5. Open browser and get into Kibana dashboard --> Wazuh app.

**Screenshots**
![image](https://user-images.githubusercontent.com/47891234/86396062-e6c99480-bcbe-11ea-9f6c-5da3dd0ee6f4.png)
![image](https://user-images.githubusercontent.com/47891234/86396241-31e3a780-bcbf-11ea-91da-3509dacb028b.png)
![image](https://user-images.githubusercontent.com/47891234/86396240-314b1100-bcbf-11ea-947e-169f20b625d2.png)

Thanks,

Arun Pandiyan P

**Comment by pablotr9** [comment]
Hi @apandiyan ,

You are right, the index pattern created should be the one specified in the `wazuh.yml`, I have just opened a new issue for that (#2354) and it will be ready for the next release. Meanwhile, as the app will create the index pattern `wazuh-alerts-3.x-*` you can manually create your custom index pattern in Kibana Management (the only requirement is to select `timestamp` as the time filter field)

Thanks for reporting it, let us know if you have any other question!

Best Regards,
Pablo Torres

**Comment by pablotr9** [comment]
Hi again @apandiyan !!

The fix for this bug has already been merged and will be included in the next release (https://github.com/wazuh/wazuh-kibana-app/pull/2356) 
I will close this issue, please feel free tp open a new issue if you have any other question, thanks!

Regards,
Pablo Torres
